### PR TITLE
pubsub docu

### PR DESCRIPTION
**Pull request by LucaMele**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,22 +57,80 @@ Do you love **Vue**? Here a helpfull link for you: https://alligator.io/vuejs/vu
...
 
-TODO.
+We have a strict strategy for releasing new versions of the Patterns Library. Please refer to the wiki: https://github.com/axa-ch/patterns-library/wiki/Crafting-a-release
+
+## Developers Guide:
 
 ### Publish / Subscribe between webcomponents
 
-TODO.
+To listen to event and triggers between component, we use some decoupled events. This logic is included in the `BaseComponent` class and is available for every component.
```
Comment (AndyOGo) [inline README.md]: Typo missing s in `To listen to events and triggers between components`

File: README.md [hunk 2]
```diff
@@ -57,22 +57,80 @@ Do you love **Vue**? Here a helpfull link for you: https://alligator.io/vuejs/vu
...
+
+The publish/subscribe system use native Custom Events (https://dom.spec.whatwg.org/#interface-customevent). Per default they don't bubble and don't cancel (no prevent default). The event name is name spaced with a slash.
```
Comment (AndyOGo) [inline README.md]: use**s**

File: README.md [hunk 3]
```diff
@@ -57,22 +57,80 @@ Do you love **Vue**? Here a helpfull link for you: https://alligator.io/vuejs/vu
...
+ * @param {String} topic - A string defining the topic to publish to.
+ * @param {*} arg - The data associate with the generated event.
+ * @param {Element} [node=document] - The node to publish message to.
+ */
+export function publish(topic, arg, node = document) {
+  ...
+}
+```
+
+The event, per default, are propagated through the document node:
```
Comment (AndyOGo) [inline README.md]: is not are

File: README.md [hunk 4]
```diff
@@ -57,22 +57,80 @@ Do you love **Vue**? Here a helpfull link for you: https://alligator.io/vuejs/vu
...
+// wants to know only when COMP B subscribed, he can listen to the Subscription event of COMP B:
+subscribe('pubsub/onsubscribe/device-state/change', () => {});
+
+```
+
+Alternatively, you can set a custom node where the event will be triggered on:
+
+![Alt text](./readme-assets/pub-sub-custom-ev.png)
+
+The concept is similar to the description above, with the difference that you can pass a context.
```
Comment (AndyOGo) [inline README.md]: I would say custom node not passing a context.
This feature is just utilised for enabling context support, but can be used by itself too.

**Comment by LucaMele** [comment]
@AndyOGo , thank you very much for your review. I updated the PR

**Pull request merged**